package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "astro-typedoc",
+  "version": "0.0.0",
+  "description": "",
+  "keywords": [],
+  "author": "Felix Khafizov <f_khafizov@protonmail.com>",
+  "license": "MIT",
+  "repository": "evilmartians/astro-typedoc",
+  "sideEffects": false,
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "type": "module",
+  "types": "./index.d.ts",
```

Comment:
You also need to remove this line too in this case.

Or you can return `index.d.ts` for types for the config.

Reply:
Removed for now